dandiapi/settings.py

```diff
@@ -89,4 +89,5 @@ class ProductionConfiguration(DandiConfig, SentryConfig, ProductionBaseConfigura
 
 
 class HerokuProductionConfiguration(DandiConfig, SentryConfig, HerokuProductionBaseConfiguration):
-    pass
+    # All login attempts in production should go straight to GitHub
+    LOGIN_URL = '/accounts/github/login/'
```

Comment:
Doesn't this need to be populated with query string parameters to start an OAuth flow?

Reply:
The default value is `/accounts/login/`, so Django just uses this string as a base and appends the appropriate query parameters. When I added this to the development settings I got redirected to:
```
http://localhost:8000/accounts/github/login/?next=/oauth/authorize/%3Fredirect_uri%3Dhttp%253A%252F%252Flocalhost%253A8085%252F%26client_id%3DDk0zosgt1GAAKfN8LT4STJmLJXwMDPbYWYzfNtAl%26response_type%3Dcode%26state%3DdDA4z1uQFE%26response_mode%3Dquery%26code_challenge%3DAiUmSqYjAdFIMCkem3CpeMuXzN_NoclrjlzTifPPUuc%26code_challenge_method%3DS256
```
Clicking on the GitHub link takes me to:
```
http://localhost:8000/accounts/github/login/?process=login&next=%2Foauth%2Fauthorize%2F%3Fredirect_uri%3Dhttp%253A%252F%252Flocalhost%253A8085%252F%26client_id%3DDk0zosgt1GAAKfN8LT4STJmLJXwMDPbYWYzfNtAl%26response_type%3Dcode%26state%3DY65zcrmYEL%26response_mode%3Dquery%26code_challenge%3D8I1AZpxXtdmmh6K0eUCFnuo-W1T-G-ujbV6SLsx2kOQ%26code_challenge_method%3DS256
```
The only differences I see are a missing query parameter `process=login` and some extra URL encoding. Sadly I don't have a way to test this without deploying it.

Reply:
I misunderstood the URL, sorry.